File: Curso_logica_de_programacao/Exercícios_logica_em_Python/Soma_impares/Soma_impares.py
x: int
y: int

print("Digite dois numeros:")
x = int(input())
y = int(input())

troca: int
if x > y:
    troca = x
    x = y
    y = troca


soma: int
soma = 0 
for i in range(x + 1, y):
    if i % 2 != 0:
        soma += i

print(f"Soma dos impares = {soma}")

File: Algoritmos_e_Logica_de_Programacao/Exercicios_logica_em_Java/Soma_impares/src/Soma_impares.java
import java.util.Scanner;
import java.util.Locale;

public class Soma_impares {
    public static void main(String[] args) throws Exception {

        Locale.setDefault(Locale.US);
        Scanner sc = new Scanner(System.in);

        System.out.println("Digite dois numeros: ");
        int x = sc.nextInt();
        int y = sc.nextInt();

        int troca;
        if (x > y) {
            troca = x;
            x = y;
            y = troca;
        }

        int soma = 0;
        for (int i = x + 1; i < y; i++) {
            if (i % 2 != 0) {
                soma += i;
            }
        }

        System.out.print("Soma dos impares = " + soma);

        sc.close();
    }
}
